client/webserver/site/src/css/main.scss

```diff
@@ -389,6 +389,7 @@ div.popup-notes {
 
 .errcolor {
   color: #e61c00;
+  word-break: break-all;
```

Comment:
Let's see a couple examples of this please.  `break-all` breaks words very readily, perhaps too readily; on the other hand, `word-break: break-word`, or `word-break: normal` with `overflow-wrap: anywhere`, would only break the words that cannot fit on one line.

Reply:
> In contrast to `word-break: break-word` and `overflow-wrap: break-word`, `word-break: break-all` will create a break at the exact place where text would otherwise overflow its container. [MDN](https://developer.mozilla.org/en-US/docs/Web/CSS/word-breakl)

`word-break: break-word` is deprecated, using `overflow-wrap: anywhere` better?

Reply:
I say we let the bootstrap project decide. https://getbootstrap.com/docs/5.0/utilities/text/#word-break